ui/components/app/signature-request-siwe/signature-request-siwe-message/index.scss

```diff
@@ -0,0 +1,34 @@
+.signature-request-siwe-message {
+  flex: 1 100%;
+  border-radius: 8px;
+  padding: 8px 16px;
+  margin: 16px;
+  border: 1px solid var(--color-border-muted);
+
+  &__list {
+    flex: 1 60%;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+
+    &__item {
+      flex: 1;
+      flex-direction: column;
+
+      &__label {
+        @include H4;
+
+        margin: 8px auto;
+      }
+
+      &__value {
+        @include H6;
+
+        margin: 8px auto;
+        white-space: pre-line;
+        overflow: hidden;
+        word-wrap: break-word;
```

Comment:
I believe white-space, overflow, and word-wrap are not cared for by our design system, so you'll need the custom css here, but you can still use the Typograpy component and remove the duplicated styles such that as the design system is extended it becomes increasingly easier to remove duplicated styles.

Reply:
Updated